packages/flutter/lib/src/cupertino/localizations.dart

```diff
@@ -384,12 +396,21 @@ class DefaultCupertinoLocalizations implements CupertinoLocalizations {
   @override
   String timerPickerHourLabel(int hour) => hour == 1 ? 'hour' : 'hours';
 
+  @override
+  List<String> get timerPickerHourLabels => const <String>['hour', 'hours'];
```

Comment:
this is a bit trickier. If you change this, you'd have to change the localization base in flutter_localizations too. As an approximate, a less exact but cheaper solution is to take a look at a few languages in https://github.com/flutter/flutter/tree/master/packages/flutter_localizations/lib/src/l10n and just query 1 or 2 values for hour/minute/seconds and query that corresponding label to get at a possible longest value. 

This wouldn't then need to get human translators to spell out all the possible variants again.

Reply:
I've been override on GlobalCupertinoLocalizations related methods, and it's working.

```dart
  @override
  List<String> get timerPickerHourLabels => <String>[
    timerPickerHourLabelZero,
    timerPickerHourLabelOne,
    timerPickerHourLabelTwo,
    timerPickerHourLabelFew,
    timerPickerHourLabelMany,
    timerPickerHourLabelOther,
  ];
```